src/openforms/submissions/tests/test_variables/test_fetch_with_logic.py

```diff
@@ -0,0 +1,110 @@
+from pathlib import Path
+
+import requests_mock
+from factory.django import FileField
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+from zgw_consumers.constants import APITypes, AuthTypes
+
+from openforms.forms.constants import LogicActionTypes
+from openforms.forms.tests.factories import FormLogicFactory
+from openforms.registrations.contrib.zgw_apis.tests.factories import ServiceFactory
+from openforms.variables.tests.factories import ServiceFetchConfigurationFactory
+
+from ..factories import SubmissionFactory
+from ..mixins import SubmissionsMixin
+
+
+class ServiceFetchWithActionsTest(SubmissionsMixin, APITestCase):
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
please call the parent method, mixins are often used to also set up some base test data used in tests